ui-kit/src/main/kotlin/org/p2p/uikit/organisms/UiKitToolbarView.kt

```diff
@@ -0,0 +1,51 @@
+package org.p2p.uikit.organisms
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.appcompat.widget.SearchView
+import androidx.core.view.isVisible
+import com.google.android.material.appbar.MaterialToolbar
+import org.p2p.uikit.R
+
+class UiKitToolbarView @JvmOverloads constructor(
```

Comment:
I guess it's ok to name it without View ending. It's toolbar after all

Reply:
ok